02-spatial-data.Rmd

```diff
@@ -108,7 +108,16 @@ knitr::include_graphics(c("figures/vector_lonlat.png", "figures/vector_projected
 ```
 
 **sf** is a package providing classes for geographic vector data.
-Not only does **sf** supersede **sp**, it also provides a consistent command-line interface to GEOS\index{GEOS} and GDAL\index{GDAL}, superseding **rgeos** and **rgdal** (described in Section \@ref(the-history-of-r-spatial)).
+Not only does **sf** supersede **sp**, it also provides a consistent command-line interface to five important low level libraries for geocomputation:
+
+- GEOS\index{GEOS}, for geometry operations such as calculating buffers and centroids, covered in Chapter \@ref(geometric-operations)
```

Comment:
I would suggest different order of the libraries: start with GDAL, then PROJ, then GEOS and S2. I am also unsure if we should list liblwgeom here, as it is not mentioned in the sf intro message...

Reply:
I suggested different order partially to keep GEOS and S2 together as they have overlapping capabilities but for data with different CRSs